### DISCUSSION TITLE
OBS 27 Version

### DISCUSSION THREAD

**Discussion by FlockersDesign** [exeldro/obs-move-transition#138] [General]
Hello i am looking for a working version for OBS 27
and because it seems the developer only releases installs for the latest versions and not the older onces and i dont uses 28 because of a lot off incompatible plugins i cant use this plugin

Thank you in advance

**Comment by exeldro** [comment]
Get version 2.6.1 or lower from
https://obsproject.com/forum/resources/move-transition.913/history

**Comment by Rilight25** [comment]
# INSTALLING PLUGINS in OBS 31.0.0 MAC M1 PRO - no success ??? SO IT WORKS: HERE IS THE SOLUTION:

# 1 COMMAND IN TERMINAL e.g. for the move-transition-PKG-file:
I need to run the code as admin (sudo). And target is just "target /"

sudo installer -pkg /Applications/OBS.app/Contents/Resources/obs-plugins/64bit/move-transition-3.1.1-macos/move-transition-3.1.1-macos-arm64.pkg -target /
installer: Package name is move-transition
installer: Upgrading at base path /        (this is upgrading because there was a file before)
installer: The upgrade was successful.

# 2 Here you find the installed plugin-files: 
/Library/Application Support/obs-studio/plugins

You have to copy them in the applications/OBS.app/Contents/Resources/obs-plugins
and thats it - they are activated in OBS

**Reply by exeldro** [reply]
Instead of the copy step you can install them directly in the correct folder `~/Library/Application Support/obs-studio/plugins`

**Comment by Rilight25** [comment]
Hi Exeldro,
Manuel install works - I have learned a lot yesterday night about terminal and install 
I have no idea why the normal install routine doesn’t work.
Maybe I will make a how-to - video about the Install steps with terminal :-)

Thanx for your support, 
Be blessed,
Matthias
